pas/PAS-44-Onboard-paras-slots.md

```diff
@@ -0,0 +1,41 @@
+---
+PAS: 0044
+title: Onboarding Parachains - Slots
+status: Proposed
+author: Alejandro (@al3mart)
+created: 27-02-2024
+---
+
+## Summary
+Means to onboard a parachain to Paseo via **leases and slot assignment**.
+This PAS is to be deprecated once parachain scheduling via coretime is deployed in Paseo.
+
+## Abstract
+In this document users can find the steps to follow to register their parachain in Paseo.
+- Maintainers of a live parachain in Polkadot or Kusama will be granted a slot in Paseo with no expiration date.
```

Comment:
Do we need to have a way to remove dead slots? (perhaps that would be a separate PAS)

An example line might be:

Parachains that have not produced a block in the last 14 days may lose their slot and need to reapply.

Reply:
Yes, definitely we should add that rule as part of the Operation and Maintenance process. I would add that task in this existing PAS:

[Paseo Testnet Process - Operation and Maintenance](https://github.com/paseo-network/paseo-action-submission/blob/main/pas/core-support-model.md)

Reply:
https://github.com/paseo-network/paseo-action-submission/issues/47

Reply:
@al3mart Let's remove this and apply what Will recommended here: https://github.com/paseo-network/paseo-action-submission/pull/44#discussion_r1511142019